.github/workflows/spread-scheduled.yaml

```diff
@@ -0,0 +1,50 @@
+name: Scheduled spread tests
+on:
+  schedule:
+    # At 03:00 on Wednesday and Sunday.
+    - cron: "0 3 * * WED,SUN"
+
+  workflow_dispatch:
+
+jobs:
+  snap-build:
+    runs-on: ubuntu-20.04
```

Comment:
Did you mean to change this to `ubuntu-latest`?

Reply:
Didn´t push the last commit! pushing it now.